src/components/EventRegistrantsModal/Modal/AddOnSpot/AddOnSpotAttendee.tsx

```
@@ -0,0 +1,237 @@
+ * />
+ * ```
+ *
+ * Uses:-
+ * - `@apollo/client` for GraphQL mutation.
+ * - `react-bootstrap` for UI components.
+ * - `react-toastify` for notifications.
+ * - `react-i18next` for translations.
+ */
+import { SIGNUP_MUTATION } from 'GraphQl/Mutations/mutations';
+import React, { useState } from 'react';
+import { Form, Button } from 'react-bootstrap';
```

ESLint

[error] 'Form' import from 'react-bootstrap' is restricted. Do not import Form directly. Use the shared FormFieldGroup component instead.

(no-restricted-imports)